### ISSUE TITLE
Deprecated Matplotlib Function "matplotlib.cm.get_cmap()" Used in "rederer.py"

### ISSUE THREAD

**Issue by otaGran** [NVlabs/sionna#444]
### Description
Viewing scenes in 3D with a coverage map will cause AttributeError when using matplotlib==3.9.0. The "matplotlib.cm.get_cmap()" function has been deprecated since matplotlib version 3.7. 

### Steps to Reproduce
Follow the [Introduction to Sionna RT](https://nvlabs.github.io/sionna/examples/Sionna_Ray_Tracing_Introduction.html) to generate a coverage map. Then, use "scene.preview(coverage_map=cm)" to view the scene with the coverage map.

```python
import os
gpu_num = 0 # Use "" to use the CPU
os.environ["CUDA_VISIBLE_DEVICES"] = f"{gpu_num}"
os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3'

# Colab does currently not support the latest version of ipython.
# Thus, the preview does not work in Colab. However, whenever possible we
# strongly recommend to use the scene preview mode.
try: # detect if the notebook runs in Colab
    import google.colab
    colab_compat = True # deactivate preview
except:
    colab_compat = False
resolution = [480,320] # increase for higher quality of renderings

# Allows to exit cell execution in Jupyter
class ExitCell(Exception):
    def _render_traceback_(self):
        pass

# Import Sionna
try:
    import sionna
except ImportError as e:
    # Install Sionna if package is not already installed
    import os
    os.system("pip install sionna")
    import sionna

# Configure the notebook to use only a single GPU and allocate only as much memory as needed
# For more details, see https://www.tensorflow.org/guide/gpu
import tensorflow as tf
gpus = tf.config.list_physical_devices('GPU')
if gpus:
    try:
        tf.config.experimental.set_memory_growth(gpus[0], True)
    except RuntimeError as e:
        print(e)
# Avoid warnings from TensorFlow
tf.get_logger().setLevel('ERROR')

tf.random.set_seed(1) # Set global random seed for reproducibility
%matplotlib inline
import matplotlib.pyplot as plt
import numpy as np
import time

# Import Sionna RT components
from sionna.rt import load_scene, Transmitter, Receiver, PlanarArray, Camera

# For link-level simulations
from sionna.channel import cir_to_ofdm_channel, subcarrier_frequencies, OFDMChannel, ApplyOFDMChannel, CIRDataset
from sionna.nr import PUSCHConfig, PUSCHTransmitter, PUSCHReceiver
from sionna.utils import compute_ber, ebnodb2no, PlotBER
from sionna.ofdm import KBestDetector, LinearDetector
from sionna.mimo import StreamManagement
# Load integrated scene
scene = load_scene(sionna.rt.scene.munich) # Try also sionna.rt.scene.etoile
# Remove old transmitter and add new one
scene.remove("tx")
scene.remove("tx")
# Configure antenna array for all transmitters
scene.tx_array = PlanarArray(num_rows=1,
                             num_cols=1,
                             vertical_spacing=0.5,
                             horizontal_spacing=0.5,
                             pattern="tr38901",
                             polarization="V")

# Configure antenna array for all receivers
scene.rx_array = PlanarArray(num_rows=1,
                             num_cols=1,
                             vertical_spacing=0.5,
                             horizontal_spacing=0.5,
                             pattern="dipole",
                             polarization="cross")

tx = Transmitter(name="tx",
                 position=[-210,73,105], # top of Frauenkirche
                 orientation=[0,0,0])
scene.add(tx)

# Create a receiver
rx = Receiver(name="rx",
              position=[45,90,1.5],
              orientation=[0,0,0])

# Add receiver instance to scene
scene.add(rx)

tx.look_at(rx) # Transmitter points towards receiver

cm = scene.coverage_map(max_depth=5,
                        diffraction=True, # Disable to see the effects of diffraction
                        cm_cell_size=(5., 5.), # Grid size of coverage map cells in m
                        combining_vec=None,
                        precoding_vec=None,
                        num_samples=int(20e6)) # Reduce if your hardware does not have enough memor
scene.preview(coverage_map=cm)


```

### Expected Behavior
A 3D preview with coverage map.

### Actual Behavior
AttributeError: module 'matplotlib.cm' has no attribute 'get_cmap'

Check the matplotlib API document [here](https://matplotlib.org/3.7.5/api/cm_api.html#matplotlib.cm.get_cmap).

### Environment
- **Sionna:** 0.17.0
- **Matplotlib:** 3.9

**Comment by SebastianCa** [comment]
Hi @otaGran,

Thanks for reporting this issue. I can reproduce the error with matplotlib==3.9.0. It will be fixed in the next version of Sionna. For the moment, please consider downgrading to matplotlib<3.9.

**Comment by merlinND** [comment]
Thank you for the head's up @otaGran, this will be fixed in the next Sionna released. In the meantime, you can either modify the usage locally or downgrade the Matplotlib version.

**Comment by jhoydis** [comment]
Fixed in v0.18.
